AirCasting/SessionViews/SessionCardView.swift

```diff
@@ -132,7 +132,9 @@ private extension SessionCardView {
             action: {
                 withAnimation {
                     isCollapsed.toggle()
-                    uiState.toggleCardExpanded(sessionUUID: session.uuid)
+                    if !(session.isFixed && !session.isFollowed) {
```

Comment:
Here. As we have this again and possibly somewhere else too in the future, could we add some extension and then maybe use session.FixedNotFollowed. That is just idea, if now please skip :)

Reply:
`unfollowedFixed` perhaps?